mava/wrappers/env_preprocess_wrappers.py

```diff
@@ -273,3 +274,145 @@ def __getattr__(self, name: str) -> Any:
             return self.__getattribute__(name)
         else:
             return getattr(self._environment, name)
+
+
+class StackObservations:
```

Comment:
if there is the possibility of either:
1. Make abstract class related to the observation that has the basics methods for example:
```
@abc.abstractmethod
 def step(self, actions: Dict) -> dm_env.TimeStep:
         """Step the environment."""
        pass
```
2. Make the StackObservations inherit from ConcatAgentIdToObservation in order to not make redundant methods like observation_spec.
But this is of course optional!

Reply:
This has been done